core-tests/src/test/java/org/mule/runtime/core/construct/processor/PipelineMessageNotificationTestCase.java

```diff
@@ -82,9 +76,16 @@
 
   private EventContext context;
 
+  @Rule
+  public ExpectedException thrown = none();
+
+  public PipelineMessageNotificationTestCase(boolean reactive) {
+    super(reactive);
+  }
+
   @Before
   public void createMocks() throws Exception {
-
+    muleContext = mock(MuleContext.class, withSettings().defaultAnswer(RETURNS_DEEP_STUBS));
```

Comment:
I wanted to reuse paramatization in abstract class, but in terms of this test implementation it works best with mock MuleContext/ServerNotification.  Can paramatize this test alternatively, but it results in duplicating code.  The other alternative is to run this test with non-mock muleContext, but thats not trivial.